dlpy/model.py

```diff
@@ -662,9 +662,6 @@ def evaluate_object_detection(self, ground_truth, coord_type, detection_data=Non
         if coord_type.lower() not in ['yolo', 'coco']:
             raise ValueError('coord_type, {}, is not supported'.format(coord_type))
 
-        self.conn.update(table=dict(name = self.model_name, where='_DLChrVal_ eq "iouThreshold"'),
```

Comment:
why we are removing this? is this related to this push?

Reply:
It is not related to this push. I just combined some fixes with the push.

Reply:
ok